core/templates/pages/feedback-updates-page/feedback-updates-page.component.spec.ts

```diff
@@ -0,0 +1,812 @@
+        expect(titleNode.innerText).toContain(expectedInnerText[index]);
+      });
+    }));
+
+    it('should sort feedback updates given sorting property as exploration' +
+      ' in ascending order', fakeAsync(() => {
+      // The default sort option for Feedback Updates is last updated.
+      expect(component.currentFeedbackThreadsSortType)
+        .toBe('lastUpdatedMsecs');
+      expect(component.isCurrentFeedbackSortDescending).toBeTrue();
+      expect(component.getValueOfFeedbackThreadSortKey())
+        .toBe('lastUpdatedMsecs');
+
+      component.setFeedbackSortingOptions('explorationTitle');
+
+      expect(component.currentFeedbackThreadsSortType)
+        .toBe('explorationTitle');
+      expect(component.getValueOfFeedbackThreadSortKey())
+        .toBe('explorationTitle');
+
+      tick();
+      fixture.detectChanges();
+
+      const feedbackListNameNodes =
+        fixture.debugElement.nativeElement
+          .querySelectorAll('.e2e-test-feedback-exploration') as
+           NodeListOf<HTMLElement>;
+
+      // The forEach loop is being used here because
+      // getValueOfSubscriptionSortKey is used in a *ngFor directive.
+      // Note that given subscription list is not sorted.
+      const expectedInnerText = ['Algebra', 'Biology', 'Three Balls', 'zebra'];
+      feedbackListNameNodes.forEach((titleNode, index) => {
+        expect(titleNode.innerText).toContain(expectedInnerText[index]);
+      });
+    }));
+
+    it('should sort feedback updates given sorting property as exploration' +
+      ' in descending order', fakeAsync(() => {
+      // The default sort option for Feedback Updates is last updated.
+      expect(component.currentFeedbackThreadsSortType)
+        .toBe('lastUpdatedMsecs');
+      expect(component.isCurrentFeedbackSortDescending).toBeTrue();
+      expect(component.getValueOfFeedbackThreadSortKey())
+        .toBe('lastUpdatedMsecs');
+
+      component.setFeedbackSortingOptions('explorationTitle');
+
+      expect(component.currentFeedbackThreadsSortType)
+        .toBe('explorationTitle');
+      expect(component.getValueOfFeedbackThreadSortKey())
+        .toBe('explorationTitle');
+
+      component.setFeedbackSortingOptions('explorationTitle');
+
+      expect(component.isCurrentFeedbackSortDescending).toBeFalse();
+
+      tick();
+      fixture.detectChanges();
+
+      const feedbackListNameNodes =
+        fixture.debugElement.nativeElement
+          .querySelectorAll('.e2e-test-feedback-exploration') as
+           NodeListOf<HTMLElement>;
+
+      // The forEach loop is being used here because
+      // getValueOfSubscriptionSortKey is used in a *ngFor directive.
+      // Note that given subscription list is not sorted.
+      const expectedInnerText = ['Zebra', 'Three Balls', 'Biology', 'Algebra'];
+      feedbackListNameNodes.forEach((titleNode, index) => {
+        expect(titleNode.innerText).toContain(expectedInnerText[index]);
+      });
```

Comment:
*Ditto

Reply:
please do the same change for all other places in other files too.

Reply:
Done PTAL